pkg/watch/registrar.go

```diff
@@ -143,6 +164,17 @@ func (r *recordKeeper) Get() vitalsByGVK {
 	return managedKinds
 }
 
+// get returns all managed vitals, merged across registrars.
+func (r *recordKeeper) get() vitalsByGVK {
+	managedKinds := make(vitalsByGVK)
```

Comment:
This looks like an abbreviated version of `get()` that doesn't do things like perform deep copies or the merge logic. That works, but we should return `len(managedKinds)` so we can keep the efficiency while also avoiding risk of future bugs from mutating referenced values.

We could then rename this to `count()` or similar

Reply:
makes sense, updated with `count()`